openunmix/predict.py

```diff
@@ -1,55 +1,11 @@
-import argparse
-
 import torchaudio
-
 from openunmix import utils
+import torch
 
 
-def inference_args(parser):
-    inf_parser = argparse.ArgumentParser(
-        description=__doc__,
-        parents=[parser],
-        add_help=True,
-        formatter_class=argparse.RawDescriptionHelpFormatter
-    )
-
-    inf_parser.add_argument(
-        '--niter',
-        type=int,
-        default=1,
-        help='number of iterations for refining results.'
-    )
-
-    inf_parser.add_argument(
-        '--wiener-win-len',
-        type=int,
-        default=300,
-        help='Number of frames on which to apply filtering independently'
-    )
-
-    inf_parser.add_argument(
-        '--residual',
-        type=str,
-        default=None,
-        help='if provided, build a source with given name'
-             'for the mix minus all estimated targets'
-    )
-
-    inf_parser.add_argument(
-        '--aggregate',
-        type=str,
-        default=None,
-        help='if provided, must be a string containing a valid expression for '
-             'a dictionary, with keys as output target names, and values '
-             'a list of targets that are used to build it. For instance: '
-             '\'{\"vocals\":[\"vocals\"], \"accompaniment\":[\"drums\",'
-             '\"bass\",\"other\"]}\''
-    )
-    return inf_parser.parse_args()
-
-
-def unmix(
-    input_file,
+def separate(
```

Comment:
the rename is fine. However, this requires changes in [`cli.py`](https://github.com/sigsep/open-unmix-pytorch/pull/65/files#diff-bbcbcc5b1b79d5796dc1c8dcd3fcdb37c8044f320622521b40ad3c40d18d95f6R138) as well.

Reply:
done